x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/LifecyclePolicy.java

```diff
@@ -241,6 +245,32 @@ public boolean isActionSafe(StepKey stepKey) {
         }
     }
 
+    /**
+     * Validate the name for an policy against some static rules. Intended to match
+     * {@link org.elasticsearch.cluster.metadata.MetaDataCreateIndexService#validateIndexOrAliasName(String, BiFunction)}
+     * @param policy the policy name to validate
+     * @throws IllegalArgumentException if the name is invalid
+     */
+    public static void validatePolicyName(String policy) {
+        if (policy.contains(",")) {
+            throw new IllegalArgumentException("invalid policy name [" + policy + "]: must not contain ','");
+        }
+        if (policy.charAt(0) == '_') {
+            throw new IllegalArgumentException("invalid policy name [" + policy + "]: must not start with '_', '-', or '+'");
+        }
+        int byteCount = 0;
+        try {
+            byteCount = policy.getBytes("UTF-8").length;
```

Comment:
I think this is cleaner: `name.getBytes(StandardCharsets.UTF_8)` and then there is no need to catch `UnsupportedEncodingException`.

Also I think that utf-8 should always be available in a Java runtime.

Reply:
Done.